### PR TITLE
Add fallback build configuration in case of no CUDA runtime library a…

### PR DESCRIPTION
**Pull request by jantonguirao**
…t the moment of nvidia-dali-tf-plugin installation

Signed-off-by: Joaquin Anton <janton@nvidia.com>

### CODE REVIEW

**Comment by jantonguirao** [comment]
Build #607534

File: dali/python/tf_plugin/setup.py.in [hunk 1]
```diff
@@ -16,21 +16,57 @@ from setuptools import setup, find_packages
...
-class CustomInstall(install, object):
-    def run(self):
-        try:
-            import tensorflow as tensorflow
-        except:
-            raise ImportError('Failed to import Tensorflow. Tensorflow must be installed before installing nvidia-dali-tf-plugin')
+def get_module_path(module_name):
+    tensorflow_path = ''
+    for d in sys.path:
+        if os.path.isdir(d):
```
Comment (jantonguirao) [inline dali/python/tf_plugin/setup.py.in]: probably I can join in one call to isdir

File: dali/python/tf_plugin/setup.py.in [hunk 2]
```diff
@@ -16,21 +16,57 @@ from setuptools import setup, find_packages
...
+            dali_lflags=" ".join(["-L" + dali_path, "-ldali"])
+    if dali_cflags is '' and dali_lflags is '':
+        raise ImportError('Could not find DALI.')
+    return (dali_cflags, dali_lflags)
 
-        tf_cflags=" ".join(tensorflow.sysconfig.get_compile_flags())
-        tf_lflags=" ".join(tensorflow.sysconfig.get_link_flags())
+class CustomInstall(install, object):
+    def run(self):
+        tf_cflags, tf_lflags = get_tf_build_flags()
```
Comment (JanuszL) [inline dali/python/tf_plugin/setup.py.in]: Indentation.

Comment (jantonguirao) [reply dali/python/tf_plugin/setup.py.in]: What is wrong with the indentation?

**Comment by JanuszL** [comment]
Rebuilds 608341

**Pull request merged**